doc/manual/source/man/dnst-key2ds.rst

```diff
@@ -0,0 +1,42 @@
+dnst-key2ds
```

Comment:
I wonder if we should write it without `-` here?

Reply:
Agreed, there is no `dnst-key2ds` command.

Reply:
Yes, for read the docs, definitetly. For the man pages it doesn't seem to make a difference (as far as I can tell from a quick test)